### DISCUSSION TITLE
Move doctrine/dbal to suggested dependencies

### DISCUSSION THREAD

**Discussion by AidasK** [spatie/laravel-health#126] [Ideas]
`"doctrine/dbal": "^3.4",` dependency should be moved to a suggested one as it's not needed for developers who are not using table size check.

**Comment by freekmurze** [comment]
Right now, this would be a breaking change for people depending on it. I'll take care of this when creating a new major version.

**Comment by onlime** [comment]
Hi @freekmurze - I don't quite get it where you need `doctrine/dbal` in this project. In [DbConnectionInfo.php#L21-L45](https://github.com/spatie/laravel-health/blob/main/src/Support/DbConnectionInfo.php#L21-L45), you are calculating table sizes for MySQL/Postgres with native SELECTs. Couldn't find any single `Doctrine\DBAL` reference either in your code. Please explain where it's used.

Now this topic gets really interesting again as it looks like we can all get rid of `doctrine/dbal` since @hafezdivandari's PR https://github.com/laravel/framework/pull/45258 was merged and shipped in Laravel v9.44 and https://github.com/laravel/framework/pull/45281 is soon going to follow.

**Comment by freekmurze** [comment]
It indeed doesn't seem to be used, removed it.
